guides/doc-Configuring_Load_Balancer/topics/Configuring_Capsule_Server_with_Default_SSL_Certificates_for_Load_Balancing_with_Puppet.adoc

```diff
@@ -68,21 +68,41 @@ root@_{smart-proxy-context}_ca.example.com_:__{smart-proxy-context}_ca.example.c
 --enable-foreman-proxy-plugin-remote-execution-ssh
 ----
 
-. On {SmartProxyServer}, generate Puppet certificates for all other {SmartProxyServer}s that you configure for load balancing, except this first system where you configure Puppet certificates signing:
+. On {SmartProxyServer}, generate Puppet certificates for all other {SmartProxyServer}s that you configure for load balancing, except the first system where you configure Puppet certificates signing:
+.. First, stop the Puppet server:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# puppet cert generate _{smartproxy-example-com}_ \
---dns_alt_names=_loadbalancer.example.com_
+# puppet resource service puppetserver ensure=stopped
 ----
 +
+.. Generate certificate for the {SmartProxyServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# puppetserver ca generate --certname _{smartproxy-example-com}_ \
+--subject-alt-names _loadbalancer.example.com_ --ca-client
+----
++
+WARNING: The '--ca-client' option in the command must only be used for server load balancers. Using the '--ca-client' option to create certificates for regular nodes can create security issues.
```

Comment:
The warning is not needed because it is stated that you must complete the procedure on the Puppet master only in the introduction: `Complete this procedure only for the system where you want to configure Smart Proxy server to generate and sign Puppet certificates for all other Smart Proxy servers that you configure for load balancing.`